backend/src/routes/api/spawner/images/index.ts

```diff
@@ -0,0 +1,15 @@
+import { KubeFastifyInstance} from '../../../../types';
+import { FastifyReply, FastifyRequest } from 'fastify';
+import { listImageStreams } from './list';
+
+module.exports = async (fastify: KubeFastifyInstance) => {
```

Comment:
Let's move this up to just `/api/images`. We don't have a spawner resource, so the images aren't a nested resource.